docs/source/daml/intro/2_Scenario.rst

```diff
@@ -105,14 +108,18 @@ To test for failing submits and keep the scenario running thereafter, or fail if
 
 ``submitMustFail`` never has an impact on the ledger so the resulting tabular scenario view just shows the two Tokens resulting from the successful ``submit`` statements. Note the new column for Bob as well as the visibilities. Alice and Bob cannot see each others' Tokens.
 
+Note, if the ``Update`` is valid but ``submitMustFail`` is used rather than ``submit``, the scenario aborts at this ``Update`` since this is in actuality a failure. 
+.. Include code example of the above statement to illustrate this aborting mechanism
+
 .. _archiving:
 
 Archiving contracts
 -------------------
+.. Archive is an Update (like create), correct? If this is the case, it’s not in the Updates tab under “Language reference docs”
```

Comment:
``archive c`` is shorthand for ``exercise c Archive``. This is explained later. If you think it would help, you can move that explanation here.